web/tests/test_views.py

```
@@ -478,20 +478,28 @@ def test_course_information_display(self):
         self.assertContains(response, self.course.subject.name)
 
     def test_session_display(self):
-        """Test that sessions are correctly displayed"""
+        """
+        Test that on the course detail page for a user who is not logged in or not enrolled:
+        - Both session titles are shown.
+        - The meeting link for the future virtual session is not visible.
+        - The in-person session location and price for the future session are visible.
+        """
         response = self.client.get(self.detail_url)
+        self.assertEqual(response.status_code, 200)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)